p5js/NeuralNetwork/NeuralNetwork_titanic/sketch.js

```diff
@@ -49,7 +43,7 @@ function predict() {
   let is_female = parseInt(select('#is_female').value());
 
   let inputs = [age, fare, is_female];
-  classifier.predict(inputs, function (err, results) {
+  nn.predict(inputs, function (err, results) {
```

Comment:
Not sure what to do here, but should we allow for `classify()` to be called instead of `predict()` when it's a "classification" task? Same function but multiple names? Or be more strict about it? Or just live with `predict()` only for DIY network?

Reply:
Yep this is resolved. I haven't separated out the {labels,confidence} but that is coming soon.